### PR TITLE
config_tools: Update Configurator UI XSD

### PR DESCRIPTION
**Pull request by amyreye**
- Update XSD files to match field names and tooltips in parameter spreadsheet

Tracked-On: #5692
Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: misc/config_tools/schema/config.xsd [hunk 1]
```diff
@@ -99,17 +99,17 @@
...
+        <xs:documentation>Enable L1 cache flush before VM entry to prevent L1 terminal fault. L1 terminal fault is a hardware vulnerability that allows unauthorized disclosure of information residing in the L1 data cache.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="MCE_ON_PSC_DISABLED" type="Boolean" default="y">
-      <xs:annotation acrn:title="MCE workaround" acrn:views="advanced">
-        <xs:documentation>Enable software workaround for Machine Check Error on Page Size Change (silicon bug in some processor families).</xs:documentation>
+      <xs:annotation acrn:title="Machine Check Error (MCE) workaround" acrn:views="advanced">
+        <xs:documentation>Enable the software workaround for Machine Check Error on Page Size Change (hardware bug in some processor families).</xs:documentation>
```
Comment (dbkinder) [inline misc/config_tools/schema/config.xsd:112]: Is that the correct wording (hardware bug)?

File: misc/config_tools/schema/config.xsd [hunk 2]
```diff
@@ -166,7 +166,7 @@
     </xs:element>
     <xs:element name="MAX_PCI_DEV_NUM" default="96">
       <xs:annotation acrn:title="Maximum number of PCI devices" acrn:views="advanced">
-        <xs:documentation>Specify the static memory allocation of maximum number of PCI devices. (Default value is 96)</xs:documentation>
+        <xs:documentation>Specify the maximum number of PCI devices. The default value is calculated from the board configuration file. If you have PCI devices that were not detected by the Board Inspector, you may need to change this maximum value.</xs:documentation>
```
Comment (dbkinder) [inline misc/config_tools/schema/config.xsd:169]: We should mention this impacts the amount of memory used to maintain information about these PCI devices.

File: misc/config_tools/schema/config.xsd [hunk 3]
```diff
@@ -193,13 +193,13 @@
...
-        <xs:documentation>The pre-defined number of interrupt sources of all pass-through devices.</xs:documentation>
+      <xs:annotation acrn:title="Maximum number of IRQ entries for passthrough devices" acrn:views="advanced">
+        <xs:documentation>Specify the maximum number of interrupt request (IRQ) entries from all passthrough devices.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="MAX_MSIX_TABLE_NUM" default="64">
-        <xs:annotation acrn:views="advanced">
-          <xs:documentation>Specify maximum number of MSI-X tables per device. If this value is empty, then the default value will be calculated from the board XML file. (Default value is 64)</xs:documentation>
+        <xs:annotation acrn:title="Maximum number of MSI-X tables per device" acrn:views="advanced">
+          <xs:documentation>Specify the maximum number of MSI-X tables per device. The default value is calculated from the board configuration file.</xs:documentation>
```
Comment (dbkinder) [inline misc/config_tools/schema/config.xsd:202]:
```suggestion
          <xs:documentation>Specify the maximum number of Message Signaled Interrupt  MSI-X tables per device. The default value is calculated from the board configuration file.</xs:documentation>
```

File: misc/config_tools/schema/config.xsd [hunk 4]
```diff
@@ -396,27 +395,27 @@ its ``id`` attribute.</xs:documentation>
...
       </xs:annotation>
     </xs:element>
     <xs:element name="os_type" type="OSType" default="Non-Windows OS">
       <xs:annotation acrn:title="OS type" acrn:applicable-vms="post-launched" acrn:views="basic">
-	<xs:documentation>Specify user VM OS type, for config tool to format the DM commandline. If it is windows, then add -windows to DM parameter. It supports Oracle virtio-blk, virtio-net and virtio-input devices for Windows guests with secure boot. For windows guest vm this option is mandatory for running Windows in a User VM. If it is not used, Windows will not recognize the virtual disk. When it is linux like OS , config tool also format the related parameters in the same way.</xs:documentation>
+	<xs:documentation>Select the OS type for this VM. This is required to run Windows in a User VM. See documentation for how to include this in the Device Model arguments.</xs:documentation>
```
Comment (dbkinder) [inline misc/config_tools/schema/config.xsd:403]: Need a link to the documentation.

File: misc/config_tools/schema/config.xsd [hunk 5]
```diff
@@ -283,12 +283,12 @@ Refer to :ref:`vuart_config` for detailed vUART settings.</xs:documentation>
     </xs:element>
     <xs:element name="vm_type" type="VMType">
       <xs:annotation acrn:title="VM type" acrn:views="basic">
-        <xs:documentation>Specify the VM type.</xs:documentation>
+        <xs:documentation>Select the VM type. A standard VM is for general-purpose applications, such as human-machine interface (HMI). A real-time VM offers special features for time-sensitive applications.</xs:documentation>
```
Comment (dbkinder) [inline misc/config_tools/schema/config.xsd:286]: We should use the actual VM Type values in this explanation:
```suggestion
        <xs:documentation>Select the VM type. A standard VM (``STANDARD_VM``) is for general-purpose applications, such as human-machine interface (HMI). A real-time VM (``RTVM``) offers special features for time-sensitive applications.</xs:documentation>
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Good start on these...

File: misc/config_tools/schema/config.xsd (hunk 1 above)
Comment (amyreye) [reply misc/config_tools/schema/config.xsd:112]: In my previous project, "silicon" was considered slang.

**Review by NanlinXie — APPROVED** [review]

**Comment by dbkinder** [comment]
@NanlinXie Why was this PR merged with unresolved change requests :(

**Pull request merged**